App/iOS/Delegates/AppDelegate.swift

```diff
@@ -118,24 +114,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 #endif
       }
 
-      let level: XCGLogger.Level = {
+      let level: OSLogType = {
         switch severity {
-        case .fatal: return .severe
+        case .fatal: return .fault
         case .error: return .error
-        case .warning: return .warning
+        // No `.warning` level exists for OSLogType. os_Log.warning is an alias for `.error`
+        case .warning: return .error
         case .info: return .info
         default: return .debug
         }
       }()
-
-      Logger.braveCoreLogger.logln(
-        level,
-        fileName: file,
-        lineNumber: Int(line),
-        // Only print the actual message content, and drop the final character which is
-        // a new line as it will be handled by logln
-        closure: { message }
-      )
+      
+      // Every brave-core log has public visibiity, otherwise nothing will be recorded.
+      let braveCoreLogger = Logger(subsystem: "com.brave.ios", category: "brave-core")
+      braveCoreLogger.log(level: level, "\(message, privacy: .public)")
```

Comment:
Shouldn't this be directed towards a security review?

Reply:
it will be decided on during the security review

Reply:
Much better to make it private imo, just in case

Reply:
If its private I don't think QA will not be able view _any_ logs from Console.app… If we are going to set it private, I'd say we only set it private for Release builds

Reply:
@kylehickinson does QA team use beta builds from testflight or do they build themselves? Private for Release and public for Beta/Debug could be ok, ideally, I would make it private for the beta if possible because beta is public.

Reply:
I set it to private for now. Exposing it as public for dev/internal builds is fine to me.

Reply:
We can also make all brave core logs to never persist and be stored in-memory by forcing it to use `debug` log level.